tests/tls-certs-alert/test.yaml

```diff
@@ -0,0 +1,20 @@
+requires:
+  min-version: 5.0.0
+  features:
+    - HAVE_LIBJANSSON
+
+args:
+ - -k none
+
+checks:
+  # Check that we have the following events in eve.json
+  # 2 - alert events
```

Comment:
2 or 3 like the actual test below?

Might be good to split the alert test into a check for each specific sig.